kratos/mpi/tests/test_distributed_sparse_matrices.py

```diff
@@ -0,0 +1,162 @@
+﻿import os
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+import KratosMultiphysics.mpi
+from KratosMultiphysics.testing.utilities import ReadModelPart
+import numpy as np
+
+def GetFilePath(fileName):
+    return os.path.dirname(os.path.realpath(__file__)) + "/" + fileName
+
+class TestDistributedSparseMatrices(KratosUnittest.TestCase):

```

Comment:
does this test run with any number of cores?
Or do we have to skip it e.g. if too many cores are used?

Reply:
it should run with any number of cores, yes